randovania/games/am2r/exporter/patch_data_factory.py

```diff
@@ -229,8 +317,11 @@ def _create_cosmetics(self):
             "etank_hud_rotation": c.etank_hud_rotation,
             "dna_hud_rotation": c.dna_hud_rotation,
             "room_names_on_hud": c.show_room_names.value,
+            "music_shuffle": _construct_music_shuffle_dict(c.music, rng),
```

Comment:
You can fix this by adding a `cosmetic_patches: AM2RCosmeticPatches` as a class attribute. It nicely overrides the generic one.